service/src/main/java/uk/gov/hmcts/reform/fpl/enums/JudgeOrMagistrateTitle.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.hmcts.reform.fpl.enums;
+
+public enum JudgeOrMagistrateTitle {
```

Comment:
Could use lombok here with a `@RequiredArgsContructor` and `@Getter` to reduce boiler plate